src/main/java/io/opentracing/contrib/tracerresolver/TracerResolver.java

```diff
@@ -55,6 +55,14 @@
      * @return The resolved Tracer or {@code null} if none was resolved.
      */
     public static Tracer resolveTracer() {
+        try { // Take care NOT to import GlobalTracer as it is an optional dependency and may not be on the classpath.
+            if (io.opentracing.util.GlobalTracer.isRegistered()) {
+                return logResolved(io.opentracing.util.GlobalTracer.get());
+            }
+        } catch (NoClassDefFoundError globalTracerNotInClasspath) {
```

Comment:
I want to add an integration tests without all optional dependencies though: #17